src/deltacode/__init__.py

```diff
@@ -106,6 +106,7 @@ def determine_delta(self):
                 except KeyError:
                     delta = Delta(100, new_file, None)
                     delta.factors.append('added')
```

Comment:
Now that we have a `status` field for the delta object, we do not need to append that string to our `factors` field.